AzureMonitorAgent/ama_tst/modules/logcollector.py

```diff
@@ -19,7 +19,15 @@
 PS_CMD_VSZ = "ps aux --sort -vsz | head -10"
 DU_CMD = "du -h -d 1 {0} /var/opt/microsoft/azuremonitoragent/events"
 VAR_DU_CMD = "du -h -d 1 {0} /var"
+LS_CMD = "ls -al {0}"
```

Comment:
please use `namei -om {0}` instead as it will check perms at every level of the tree (will help if parent directories are wrong perms/owners)

Reply:
Modified PR to add ```namei``` cmd to check directory permissions and keep this ```ls``` cmd to list the file permissions